### PR TITLE
ISSUE #2635 allow unity to rerun if it was terminated mid operation

### PR DESCRIPTION
**Pull request by carmenfan**
See https://github.com/3drepo/3drepounity/pull/387 for details

### CODE REVIEW

File: .azure/build-and-deploy.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ variables:
   branchName: master
   group: tests-group
   customHelmOverride: |
-    config.CUSTOM=example
+    config.APP_QUEUE_DNS=issue-2635-rabbitmq
```
Comment (Charence) [inline .azure/build-and-deploy.yaml:34]: is this a temporary config for testing or we want to keep it as this value?

Comment (carmenfan) [reply .azure/build-and-deploy.yaml:34]: this file is under some special setting where any changes are ignored when we merge i believe. @keifgwinn ?

Comment (keifgwinn) [reply .azure/build-and-deploy.yaml:34]: https://github.com/3drepo/3drepo.io/blob/1d8de93c45a426bcd8c84008cef4240bb5f4abe7/.gitattributes#L8 is supposed to ignore changes to this file when doing the merge, however I noticed it didn't work last time.

Comment (carmenfan) [reply .azure/build-and-deploy.yaml:34]: sounds like this is only triggered if there's a conflict https://stackoverflow.com/a/27234774

so it wouldn't work if the staging version was not touched as well.

Comment (keifgwinn) [reply .azure/build-and-deploy.yaml:34]: regarding the original merge, this file can be abandoned it's only for moving it to a dedicated queue.

Comment (carmenfan) [reply .azure/build-and-deploy.yaml]: @Charence reverted

**Review by Charence — APPROVED** [review]

**Pull request merged**